docs/contribute.rst

```diff
@@ -29,13 +29,37 @@ couple of recommendations to keep on mind when writing code:
 * When using hanging indent, the first line should be empty.
 * The closing brace/bracket/parenthesis on multiline constructs
   is under the first non-whitespace character of the last line.
+* When generating user messages use the whole sentence with the
+  first word capitalized and enclose any names in single quotes:
 
-When generating user messages use the whole sentence with the
-first word capitalized and enclose any names in single quotes:
+  .. code-block:: python
 
-.. code-block:: python
+     self.warn(f"File '{path}' not found.")
 
-    self.warn(f"File '{path}' not found.")
+* For multiline shell script, use multiline strings - :py:class:`ShellScript`
```

Comment:
```suggestion
* For multiline shell scripts, use multiline strings - :py:class:`ShellScript`
```

Just a nitpick: Otherwise `For a multiline shell script` should be used?